pkg/storage/gcp/getter.go

```diff
@@ -14,7 +14,7 @@ import (
 // The caller is responsible for calling close on the Zip ReadCloser
 func (s *Storage) Get(module, version string) (*storage.Version, error) {
 	ctx := context.Background()
-	if exists := s.Exists(module, version); !exists {
+	if exists := s.Exists(context.TODO(), module, version); !exists {
```

Comment:
why not pass the paxx the `ctx` value from one line above? You can change it from `Background` to `TODO` if you'd like.

Reply:
Done. Good catch!